core/evancore.vue.libs/src/i18n/de.json

```
@@ -110,6 +110,7 @@
     },
     "need-an-account": "Noch kein Account? <a href=\"#/dashboard.vue.evan/onboarding.vue.evan/sign-up\">Account erstellen</a>",
     "no": "Nein",
+    "no-permission": "Keine Berechtigung zur Anzeige",
```

Sounds a bit weirdo. Maybe something like this?

```suggestion
    "no-permission": "Sie sind nicht berechtigt den Inhalt einzusehen.",
```